File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/07 - Nested Conditional Statements Exercise/02. Cinema .py
type_move = input()
r = int(input())
c = int(input())

total_seats = r * c
price = 0

if type_move == "Premiere":
    price = 12.00 * total_seats
elif type_move == "Normal":
    price = 7.50 * total_seats
elif type_move == "Discount":
    price = 5.00 * total_seats

print(f"{price:.2f} leva")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/10. УПРАЖНЕНИЕ ПО-СЛОЖНИ ПРОВЕРКИ/02. Cinema.js
function cinema(input) {
    let projectionType = input.shift();
    let rows = Number(input.shift());
    let cols = Number(input.shift());

    let price = 0;

    switch(projectionType) {
        case "Premiere":
            price = rows * cols * 12; break;
        case "Normal":
            price = rows * cols * 7.5; break;
        case "Discount":
            price = rows * cols * 5.00; break;
    }

    console.log(`${price.toFixed(2)} leva`);
}

cinema(['Premiere', '10', '12']);  // Expected Output: 1440.00 leva
cinema(['Normal', '21', '13']);  // Expected Output: 2047.50 leva
cinema(['Discount', '12', '30']);  // Expected Output: 1800.00 leva